modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMUtils.macro

```diff
@@ -64,4 +64,13 @@ definition {
 		CreateObject.saveObject();
 	}
 
+	macro viewRoles {
+		for (var roleName : list "${roleNameList}") {
+			AssertTextEquals(
+				key_locationName = "${roleName}",
+				locator1 = "ItemSelector#LOCATION_INFO_LIST_TITLE",
+				value1 = "${roleName}");
+		}
+	}
```

Comment:
Hi @heiditrindade, I think that you could add a search on this macro. I saw that the 'roles' section, on DXP, have a lot of another roles, so it's more secure if before assert the roles, search for them. What do you think?

Reply:
Hi @AfonsoHSantos! I'll do this! Thanks for your suggestion!